Formula/kubernetes-cli.rb

```diff
@@ -12,26 +12,16 @@ class KubernetesCli < Formula
     sha256 "808c93e78b84f743877690d72aee24a61415af5fc1cc4c7bbd6c7b60fa66b735" => :yosemite
   end
 
-  devel do
-    # building from the tag lets it pick up the correct version info
-    url "https://github.com/kubernetes/kubernetes.git",
-        :tag => "v1.4.0-alpha.3",
-        :revision => "b44b716965db2d54c8c7dfcdbcb1d54792ab8559"
-    version "1.4.0-alpha.3"
-  end
-
   depends_on "go" => :build
 
   def install
-    if build.stable?
-      system "make", "all", "WHAT=cmd/kubectl", "GOFLAGS=-v"
-    else
-      # avoids needing to vendor github.com/jteeuwen/go-bindata
-      rm "./test/e2e/framework/gobindata_util.go"
+    # Patch needed to avoid vendor dependency on github.com/jteeuwen/go-bindata
+    # Build will otherwise fail with missing dep
+    # Raised in https://github.com/kubernetes/kubernetes/issues/34067
+    rm "./test/e2e/framework/gobindata_util.go"
+
+    system "make", "all", "WHAT=cmd/kubectl", "GOFLAGS=-v"
 
-      ENV.deparallelize { system "make", "generated_files" }
```

Comment:
I think this may still be needed: https://bot.brew.sh/job/Homebrew%20Core/8880/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_kubernetes_cli/

Is that what you were seeing, @ilovezfs? Did you report it upstream? Thanks!